src/components/Calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './Calculator.css';
+import InputText from './InputText';
+
+const Calculator = () => (
+  <section className="calculator_container">
+    <form action="">
+      <div>
+        <InputText />
+      </div>
+
+      <article className="calcButtons_container">
+        <div className="buttons">
+          <input type="button" value="AC" />
+          <input type="button" value="+/-" />
+          <input type="button" value="%" />
+          <input className="operators" type="button" value="&divide;" />
+        </div>
+        <div className="buttons">
+          <input type="button" value="7" />
+          <input type="button" value="8" />
+          <input type="button" value="9" />
+          <input className="operators" type="button" value="&times;" />
+        </div>
```

Comment:
[OPTIONAL] 

- [ ] I want to kindly advice that you create a single reuseable component for these input fields, one of the main goal of react is sto help you write reusable code and this is one of the places where you can achieve that :+1: